app/assets/stylesheets/dlme.scss

```diff
@@ -21,7 +22,7 @@
 }
 
 body {
-  background-image: asset-url("dlme-background.png");
```

Comment:
Should we remove the file as well?

Reply:
Yep, thanks for catching that. Just updated the PR.